### PR TITLE
Events-Updating-July-USWDS-Call-Page

### PR DESCRIPTION
**Pull request by DanielJPino-ux**
removing registration url and captioning link

**Summary**

This PR implements the following **changes:**

* [ ] change 1

**URL / Link to page**

### CODE REVIEW

**Comment by DanielJPino-ux** [comment]
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/June-2020-USWDS-Call-Update/event/2020/07/16/uswds-monthly-call-july-2020/ 

NOTE: if possible can we eliminate the "register" button from the page?

**Comment by saracope** [comment]
The time changed on this one too. We should put this on the GitHub office hours agenda to take a look at what’s happening with these. @DanielJPino-ux

File: content/events/2020/06/2020-06-18-uswds-monthly-call-june-2020.md [hunk 1]
```diff
@@ -1,42 +1,32 @@
...
 host: "U.S. Web Design System"
 event_organizer: "Digital.gov"
-registration_url: https://www.eventbrite.com/e/uswds-monthly-call-june-2020-registration-96967447245
-captions: https://www.captionedtext.com/client/event.aspx?EventID=4340563&CustomerID=321
+registration_url: 
+captions: 
 
 # start date
-date: 2020-06-18 14:30:00 -0500
+date: 2020-06-18 15:30:00 -0500
```
Comment (saracope) [inline content/events/2020/06/2020-06-18-uswds-monthly-call-june-2020.md:14]: It looks like the time changed here also.

File: content/events/2020/06/2020-06-18-uswds-monthly-call-june-2020.md [hunk 2]
```diff
@@ -1,42 +1,32 @@
...
 
 # end date
-end_date: 2020-06-18 15:30:00 -0500
-
-# see all topics at https://digital.gov/topics
-topics:
```
Comment (saracope) [inline content/events/2020/06/2020-06-18-uswds-monthly-call-june-2020.md:21]: The topics and author should stay right?

**Comment by ToniBonittoGSA** [comment]
Added HOLD b/c I don't know how to fix branch conflicts, and I can't tell if this https://digital.gov/event/2020/07/16/uswds-monthly-call-july-2020/ still needs any edits.

**Comment by DanielJPino-ux** [comment]
thanks @ToniBonittoGSA - we are getting the updated page language today i'll update this PR so we can close the other outlier and focus solely on commits to this page

**Pull request closed without merging**